ocs_ci/utility/openshift_dedicated.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf8 -*-
+"""
+Module for interactions with Openshift Dedciated Cluster.
+
+"""
+
+
+import logging
+import os
+import json
+
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import run_cmd, exec_cmd
+from ocs_ci.utility import utils
+
+logger = logging.getLogger(name=__file__)
+openshift_dedicated = config.AUTH.get("openshiftdedicated", {})
+
+
+def login():
+    """
+    Login to OCM client
+    """
+    token = openshift_dedicated["token"]
+    cmd = f"ocm login --token={token} --url=staging"
+    logger.info("Logging in to OCM cli")
+    run_cmd(cmd, secrets=[token])
```

Comment:
you can use exec_cmd instead as we are trying to depricate run_cmd and rather use exec_cmd, but I will not block PR on that.

Reply:
Noted, will use exec_cmd in future cases.